### PR TITLE
Feat 62 burrholes

### LINKED ISSUE #62: Map 2023 Burr Holes
# User story
As a developer, I want to map Sharepoint 2023 response to multiple Injections/Fiber Implants based on Burr Hole fields

# Acceptance criteria
- [x] Create a method that loops through burr holes and accordingly maps them to Injections/Fiber Implants
- [x] Map Burr hole 1-4 to Injections
- [x] Map Burr hole 1-4 to Fiber Implants
- [x] upgrade aind-data-schema to include ProbeD name

# Sprint Ready Checklist 
- [x] 1. Acceptance criteria defined
- [x] 2. Team understands acceptance criteria
- [x] 3. Team has defined solution / steps to satisfy acceptance criteria
- [x] 4. Acceptance criteria is verifiable / testable
- [x] 5. External / 3rd Party dependencies identified
- [x] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

### PR DESCRIPTION
**Pull request by mekhlakapoor**
closes #62

### CODE REVIEW

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
aind-data-schema should only be in the server dependencies

File: src/aind_metadata_service/sharepoint/client.py [hunk 1]
```diff
@@ -1295,42 +1308,559 @@ def _map_list_item_to_head_frame(
...
+                        injection_hemisphere=burr_hemisphere,
+                        injection_coordinate_ml=burr_coordinate_ml,
+                        injection_coordinate_ap=burr_coordinate_ap,
+                        injection_coordinate_depth=burr_coordinate_depth,
+                        injection_angle=burr_angle,
+                        injection_volume=injection_volume,
+                    )
+                burr_4_procedures.append(injection)
+            elif procedure.strip() == nsb_burr_types.FIBER_IMPLANT.value:
+                name = ProbeName.PROBE_D.value
```
Comment (jtyoung84) [inline src/aind_metadata_service/sharepoint/client.py:1842]: Probe_D isn't defined anywhere

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
There's an issue with calling one of the enums

File: src/aind_metadata_service/sharepoint/client.py (hunk 1 above)
Comment (mekhlakapoor) [reply src/aind_metadata_service/sharepoint/client.py:1842]: it's defined in aind-data-schema 0.12.13, did you run pip install -e .[dev]

Comment (jtyoung84) [reply src/aind_metadata_service/sharepoint/client.py:1842]: My bad, it looks like it's defined in the updated aind-data-schema

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**